examples/dynamo/refit_engine_example.py

```diff
@@ -0,0 +1,136 @@
+"""
+.. _refit_engine_example:
+
+Refit  TenorRT Graph Module with Torch-TensorRT
+===================================================================
+
+We are going to demonstrate how a compiled TensorRT Graph Module can be refitted with updated weights.
+
+In many cases, we frequently update the weights of models, such as applying various LoRA to Stable Diffusion or constant A/B testing of AI products.
+That poses challenges for TensorRT inference optimizations, as compiling the TensorRT engines takes significant time, making repetitive compilation highly inefficient.
+Torch-TensorRT supports refitting TensorRT graph modules without re-compiling the engine, considerably accelerating the workflow.
+
+In this tutorial, we are going to walk through
+1. Compiling a PyTorch model to a TensorRT Graph Module
+2. Save and load a graph module
+3. Refit the graph module
+"""
+
+# %%
+# Standard Workflow
+# -----------------------------
+
+# %%
+# Imports and model definition
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+import torch_tensorrt as trt
+import torchvision.models as models
+from torch import nn
+from torch_tensorrt.dynamo._refit import refit_module_weights
```

Comment:
Lets add `torch_tensorrt.dynamo._refit.refit_module_weights` to `torch_tensorrt/dynamo/__init__.py`

So users can do `torch_tensorrt.dynamo.refit_module_weights`